.circleci/config.yml

```
@@ -284,11 +281,13 @@ jobs:
 
   truffle-sample-project:
     docker:
-      - image: circleci/node:16
+      - image: cimg/node:16.15
     steps:
       - update-npm
       - show-npm-version
       - provision-and-package-solcjs
+      - run: sudo apt update
+      - run: sudo apt install python3 --assume-yes --no-install-recommends
```

```suggestion
      - run: sudo apt install python3 python-is-python3 --assume-yes --no-install-recommends
```
might fix the sqlite thing I see in CI?

---

That worked. Thanks.

I also had to do another fix - Truffle updated its `metacoin` box and it seems to work a bit differently. A small config tweak solved it though. Now the job passes.